lib/gssapi/mech/gss_oid.c

```diff
@@ -151,6 +151,9 @@ gss_OID_desc GSSAPI_LIB_VARIABLE __gss_c_ntlm_reset_crypto_oid_desc = { 11, rk_U
 /* GSS_NEGOEX_MECHANISM - 1.3.6.1.4.1.311.2.2.30 */
 gss_OID_desc GSSAPI_LIB_VARIABLE __gss_negoex_mechanism_oid_desc = { 10, rk_UNCONST("\x2b\x06\x01\x04\x01\x82\x37\x02\x02\x1e") };
 
+/* GSS_SANON_X25519_MECHANISM - 1.3.6.1.4.1.5322.26.1.110 */
+gss_OID_desc GSSAPI_LIB_VARIABLE __gss_sanon_x25519_mechanism_oid_desc = { 10, rk_UNCONST("\x2b\x06\x01\x04\x01\xa9\x4a\x1a\x01\x6e") };
```

Comment:
This reminds me: we need some tool to test that all these hard-coded DER OID byte strings are correct.  I'm not sure how to build such a tool.  Perhaps we could have a text file with `varname OID` with the OID in dotted form, and then "compile" it to a `.c` file, then we'd not need any test of correctness here.

Reply:
Oh, what are the OID arc names?

Reply:
Ahh: `iso(1) org(3) dod(6) internet(1) private(4) enterprise(1) padl(5322)`.

Reply:
Oh, never mind.  This _is_ generated, but also committed.